packages/grid/_modules_/grid/hooks/features/selection/useGridSelection.ts

```diff
@@ -25,6 +25,7 @@ import { GridPreProcessingGroup, useGridRegisterPreProcessor } from '../../core/
 import { GridCellModes } from '../../../models/gridEditRowModel';
 import { GridColumnsRawState } from '../columns/gridColumnsState';
 import { isKeyboardEvent } from '../../../utils/keyboardUtils';
+import { gridClasses } from '../../..';
```

Comment:
We should probably not import it from the parent index file to avoid risk of circular imports

Reply:
VSCode import suggestions never work correctly.

Reply:
Neither do Webstorm one :/